.openpublishing.redirection.json

```diff
@@ -418,6 +430,38 @@
             "source_path": "docs/csharp/language-reference/keywords/interpolated-strings.md",
             "redirect_url": "/dotnet/csharp/language-reference/tokens/interpolated"
         },
+        {
+            "source_path": "docs/csharp/language-reference/keywords/iteration-statements.md",
+            "redirect_url": "/dotnet/csharp/language-reference/keywords/statement-keywords"
+        },
+        {
+            "source_path": "docs/csharp/language-reference/keywords/jump-statements.md",
+            "redirect_url": "/dotnet/csharp/language-reference/keywords/statement-keywords"
+        },
+        {
+            "source_path": "docs/csharp/language-reference/keywords/literal-keywords.md",
+            "redirect_url": "/dotnet/csharp/language-reference/keywords"
```

Comment:
The same comment applies here -- perhaps link to `null` rather than the general list of statements.

Reply:
Also in the [proposed TOC](https://gist.github.com/BillWagner/047cb97f6c0fe542d3a1299e994b2acd#-literal-constants)